### PR TITLE
77 integrate middleware and allow to query aens auction related information

### LINKED ISSUE #64: ÆNS: auction-related functionalities
calculation of the initial `nameFee` is already implemented.

there are still open topics regarding auctions. if a name is already claimed we need to provide a way to:
   - [x] calculate the next minimum fee if the claim is not final
   - [x] determine if the claim is final and if not return the block-height which will make the claim final

### LINKED ISSUE #77: integrate middleware and allow to query ÆNS auction related information
Tasks:
- [x] integrate middleware in docker-compose
   - **note:** the new auction related endpoints aren't available in the current middleware-release. therefore we temporarily need to build our own docker-image to make use of the new endpoints
- [x] provide a middleware-service to perform ÆNS related queries

### PR DESCRIPTION
**Pull request by marc0olo**
closes #77 
closes #64

### CODE REVIEW

File: src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java [hunk 1]
```diff
@@ -30,6 +30,9 @@
   static String oracleId;
   static String queryId;
 
+  static String queryString = "Am I stupid?";
+  static String responseString = "yes you are nuts!";
```
Comment (mitch-lbw) [inline src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java]: please use a more senseful test string

File: src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java [hunk 2]
```diff
@@ -132,22 +130,35 @@ public void dOracleRespondTest(TestContext context) {
         context,
         t -> {
           try {
+            OracleQueriesResult oracleQueriesResult =
+                this.aeternityServiceNative.oracles.blockingGetOracleQueries(
+                    oracleAccount.getPublicKey().replace("ak_", "ok_"),
```
Comment (mitch-lbw) [inline src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java]: a discussed refactor and extract to BaseKeyPair or Util class

Comment (marc0olo) [reply src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java]: added methods in the BaseKeyPair class

File: src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java (hunk 1 above)
Comment (marc0olo) [reply src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java]: now using an example query and response from https://api.openweathermap.org/

**Pull request merged**